traffic_ops/testing/api/v4/coordinates_test.go

```diff
@@ -147,6 +158,35 @@ func SortTestCoordinates(t *testing.T) {
 	}
 }
 
+func SortTestCoordinatesDesc(t *testing.T) {
+
+	respAsc, _, err1 := TOSession.GetCoordinates(nil, nil)
+	params := url.Values{}
+	params.Set("sortOrder", "desc")
+	respDesc, _, err2 := TOSession.GetCoordinates(params, nil)
+
+	if err1 != nil {
+		t.Errorf("Expected no error, but got error in Coordinates Ascending %v", err1)
+	}
+	if err2 != nil {
+		t.Errorf("Expected no error, but got error in Coordinates Descending %v", err2)
+	}
+
+	if len(respAsc) > 0 && len(respDesc) > 0 {
+		// reverse the descending-sorted response and compare it to the ascending-sorted one
+		for start, end := 0, len(respDesc)-1; start < end; start, end = start+1, end-1 {
+			respDesc[start], respDesc[end] = respDesc[end], respDesc[start]
+		}
+		if respDesc[0].Name != "" && respAsc[0].Name != "" {
+			if !reflect.DeepEqual(respDesc[0].Name, respAsc[0].Name) {
+				t.Errorf("Coordinates responses are not equal after reversal: %s - %s", *&respDesc[0].Name, *&respAsc[0].Name)
```

Comment:
nit, but you don't need `reflect.DeepEqual` to check string equality, and `*&` is a no-op

Reply:
Fixed